.circleci/config.yml

```diff
@@ -97,6 +97,10 @@ jobs:
             cd docs
             make html
 
+      - store_artifacts:
+          name: store documentation
+          path: ./docs/build
```

Comment:
This is very useful. I didn't know that we could preview documentations on CircleCI. This is not a strong opinion but this change could actually be separated into it's own PR.

Reply:
Got it. I'll revert this change and open a separate RP for this change.